sr_example/scripts/sr_example/hand_examples/sr_right_hand_follow_named_trajectory.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+# This example demonstrates how you can send a trajectory created from named poses.
+
+import rospy
+
+from sr_robot_commander.sr_hand_commander import SrHandCommander
+
+rospy.init_node("named_traj_example", anonymous=True)
+rospy.sleep(1)  # Do not start at time zero
+
+
+hand_commander = SrHandCommander()
```

Comment:
We need to use the hand finder to properly instantiate the hand commander (in a generic way)